### ISSUE TITLE
[shelly] Shelly Pro 3EM is missing accumulated (total) channels

### ISSUE THREAD

**Issue by leifbladt** [openhab/openhab-addons#16552]
I have a Shelly Pro 3EM in (default) tri-phase profile and since firmware version 1.1 there are three channels **total_current**, **total_act_power**, and **total_aprt_power**, which are not visible in openHAB.

## Expected Behavior
I would expect that all channels (especially **total_current**, **total_act_power**, and **total_aprt_power**) described in the [documentation](https://shelly-api-docs.shelly.cloud/gen2/ComponentsAndServices/EM#status) are shown, when the Pro EM is used in **tri-phase** profile.

## Current Behavior
Only the single phase channels are shown.

## Possible Solution
There seems to be support for the accumulated channels, but they are only created when more than one meter is available. In the tri-phase profile there is only meter and the phases are prefixed with a_, b_, and _c (like c_current or a_act_power).
See https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/provider/ShellyChannelDefinitions.java#L330

## Your Environment
* openHAB 4.1.1, Docker

**Comment by leifbladt** [comment]
I can confirm that the three mentioned values are transferred to the binding, when switching the log level of the Shelly binding to `DEBUG`.

> NotifyStatus update received: {"src":"shellypro3em-3ce90e6f8cf4","dst":"shellypro3em-1de8dd8f9e","method":"NotifyStatus","params":{"ts":1.71097470421E9,"em:0":{"id":0,"a_current":0.811,"a_voltage":236.6,"a_act_power":35.0,"a_aprt_power":192.0,"a_pf":0.55,"b_current":0.523,"b_voltage":237.0,"b_act_power":21.1,"b_aprt_power":124.1,"b_pf":0.55,"c_current":0.777,"c_voltage":236.9,"c_act_power":83.5,"c_aprt_power":183.9,"c_pf":0.65,**"total_current":2.11,"total_act_power":139.678,"total_aprt_power":500.052**}}}
```

**Comment by the-gangster** [comment]
I don't own the Pro 3EM, but wasn't one difference to the "old" Shelly 3EM (which I have), that it does not have a relay?
So I assume it fails to add the accumulated channels due to the "hasRelays"-check:
boolean accuChannel = profile.hasRelays && profile.numMeters > 1 && !profile.isRoller && !profile.isRGBW2;

But I should mention I only looked at those few lines of code that your link kicked me to. So I could completely misunderstand it.

**Comment by leifbladt** [comment]
Thanks @the-gangster, I checked the specs for the three mentioned devices (Pro 3EM, Pro EM, and 3EM): the Pro 3EM has no built in relay, it's only available as add-on.

I could get the configuration of the thing, which confirms your assumptions. I misinterpreted the `numMeter`btw 😄 
```
Device hasRelays:false (numRelays=0),isRoller:false (numRoller=0),isDimmer:false,numMeter=3,isEMeter:true)
```

I will try to build and test an updated version, to see if we're at the right spot here.
